src/bioregistry/data/bioregistry.json

```diff
@@ -88335,6 +88335,21 @@
           }
         ],
         "uri_format": "https://seq2fun.dcmb.med.umich.edu/furna/pdb.cgi?pdbid=$1"
+      },
+      {
+        "code": "psskb",
+        "description": "PSSKB allows researchers to model post translational modifications and amino acid substitutions in proteins, providing 3D structural representations. This tool is important for studying how modifications impact protein stability, localization, and function.",
+        "homepage": "https://psskb.org/",
+        "name": "Protein Structure and Stability Knowledge Base",
+        "publications": [
+          {
+            "doi": "10.1002/jcc.70046",
+            "pubmed": "39876062",
+            "title": "PSSKB: A Web Application to Study Protein Structures",
+            "year": 2025
+          }
+        ],
+        "uri_format": "https://www.rcsb.org/structure/$1"
```

Comment:
The `uri_format` here would have to be a PSSKB URL, not a PDB URL. Same for UniProt below.